modules/backendscheduler/backendscheduler.go

```diff
@@ -0,0 +1,391 @@
+package backendscheduler
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"sort"
+	"time"
+
+	"github.com/go-kit/log/level"
+	"github.com/google/uuid"
+	"github.com/grafana/dskit/services"
+	"github.com/grafana/tempo/modules/backendscheduler/work"
+	"github.com/grafana/tempo/modules/overrides"
+	"github.com/grafana/tempo/modules/storage"
+	"github.com/grafana/tempo/pkg/tempopb"
+	"github.com/grafana/tempo/pkg/util/log"
+	"github.com/grafana/tempo/tempodb/blockselector"
+	"github.com/jedib0t/go-pretty/v6/table"
+)
+
+// var tracer = otel.Tracer("modules/backendscheduler")
+
+// BackendScheduler manages scheduling and execution of backend jobs
+type BackendScheduler struct {
+	services.Service
+
+	cfg       Config
+	store     storage.Store
+	overrides overrides.Interface
+
+	work *work.Queue
```

Comment:
these don't seem to be persisted anywhere. we need to do that in the initial impl

Reply:
This has been updated to flush and load the work from the backend.